### PR TITLE
Task/3488/create whydcl page

### PR DESCRIPTION
**Pull request by geethamondi**
This PR adds a new "Why Data Center Light Page?" and has an updated menu.
Please refer to redmine task #3488 for details.

Please review this @pcoder @verysanghee

### CODE REVIEW

**Comment by verysanghee** [comment]
@geethamondi typo:  Actions speak louder words.-> Actions speak louder than words.

**Comment by verysanghee** [comment]
@geethamondi you can designate reviewers on the right. I did it just now ;) next time you can do it.

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        <div class="container">
+            <div class="row">
+                <div class="col-xs-12 col-sm-6 col-md-6">
+                    <div class="split-text">
+                        <div class="split-title">
+                            <h2>{% trans "Tech Stack" %}</h2>
+                        </div>
+                        <div class="split-description">
+                            <h3>{% trans "We are seriously open source." %}</h3>
+                            <p class="lead">{% blocktrans %} Our full software stack is opensource-We don't use anything that isn't opensource. <br>Yes we are that cool. {% endblocktrans %}</p>
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: is opensource – we
Yes, we are

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+					<div class="col-xs-12 col-sm-12 col-md-6 col-md-6 logo-wrap">
+						<img class="img-responsive" src="{% static 'datacenterlight/img/cdistbyungleich.png' %}" alt="Cdist byu ngleich">
+					</div>
+					<div class="col-xs-12 col-sm-12 col-md-6 col-md-6 logo-wrap">
+						<img class="img-responsive" src="{% static 'datacenterlight/img/prometheus.png' %}" alt="Prometheus">
+					</div>
+                </div>
+                <div class="col-xs-12 col-sm-8 col-md-6 text-right">
+                        <div class="tech-sub-sec">
+                            <h2>{% trans "We believe in giving back to FOSS community." %}</h2>
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: to the FOSS community

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                            <p class="lead">{% blocktrans %}Data Center Light is the child of free and open source(FOSS). <br>We grew up with it, live by it, and belive in it.<br> The more we work on our data center,<br> the more we contribute back to the FOSS community.{% endblocktrans %}</p>
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: free and open source software (FOSS)

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+        <hr class="thick-divider"/><!-- Divider -->
+        <div class="space">
+        <div class="container container-small">
+            <div class="row">
+				<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+					<div class="col-xs-12 col-sm-7 col-md-6 col-lg-7 text-left tech-sub-sec ">
+						<div class="tech-sub-sec">
+							<h2>{% trans "We bring the future to you" %}</h2>
+							<p class="lead">{% blocktrans %} Data Center Light uses the most modren technologies out there.<br>
+							Your VM needs only IPv6. Data Center Light provide<br> transparent two-way IPv6/IPv4 translation.
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: provides

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+					</div>
+				</div>
+				<div class="col-lg-12 space-block"></div>
+				<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+					<div class="col-xs-12 col-sm-7 col-md-6 col-lg-6 percent-text">
+							<span class="pull-left space-middle"> 100%<strong>SSD</strong></span> <span class="pull-left"><img class="img-responsive" src="{% static 'datacenterlight/img/ssd.jpg' %}" alt="SSD"></span>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-6 col-lg-6 text-right tech-sub-sec  padding-vertical">
+						<div>
+							<p class="lead">{% blocktrans %} No more hard disks with spinning metal plates!<br>
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: I would just say "No more spinning metal plates!" -- that sounds better to me.

File: datacenterlight/templates/datacenterlight/whydatacenterlight.html [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+    </div>
+        <!-- /.content-section-b -->
+    <div class="content-section-a pricing-section" id="price">
+
+        <div class="container">
+        <!-- Page Features -->
+            <div class="row text-center">
+                <div class="col-xs-12 col-md-6 text">
+                    <h2 class="section-heading">{% trans "Starting from only 15CHF per month. Try now." %}</h2>
+                    <p class="lead">{% trans "Actions speak louder words. Let's do it, try our VM now." %}</p>
```
Comment (AnotherKamila) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: louder than words

Comment (verysanghee) [inline datacenterlight/templates/datacenterlight/whydatacenterlight.html]: this I mentioned above too

**Comment by verysanghee** [comment]
"We bring the future to you" section needs to be aligned with other left-aligned sections (tech stack & starting from 15CHF..)

**Pull request merged**